README.md

```diff
@@ -22,7 +22,7 @@ Also see our [`Good for community contribution`](https://github.com/angular/mate
 ## The goal of Angular Material
 Our goal is to build a set of high-quality UI components built with Angular 2 and TypeScript, 
 following the Material Design spec, that are consumable by both JavaScript and Dart. These 
-components will serve as an example of how to write Angular code following best practices.
+components will serve as an example of how to write Angular code following [best practices](https://github.com/mgechev/angular2-style-guide).
```

Comment:
I'd rather not link to this doc at this time; best practices are still forming and the team is working on a canonical guide.